test/test_torrents/duplicate_files2.torrent

```diff
@@ -0,0 +1 @@
hi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteed6:lengthi5e4:pathl3:foo7:bar.txteed6:lengthi4e4:pathl3:foo7:bar.txteee4:name4:temp12:piece lengthi16384e6:pieces20:aaaaaaaaaaaaaaaaaaaaee
```

Comment:
this file need to be included in the tarball, and is it added to the fuzzing corpus?